docs/user/glossary.rst

```diff
@@ -53,13 +53,13 @@ Forbidden translations
 
 .. versionadded:: 4.5
 
-You can flag certain glossary terms as forbidden - those should not be used
-while translating. Use this to clarify translation when some words are
+You can flag certain glossary terms as forbidden, meaning ones _not_ be used
```

Comment:
I'm not sure but this one sounds a bit wrong... At least there is something wrong with the grammar. A bare minimum would be:

```suggestion
You can flag certain glossary terms as forbidden, meaning ones are _not_ to be used
```

But I'd suggest to change it to something like this:
```suggestion
You can flag certain glossary terms as forbidden. This will indicate that the translations of those should _not_ be used.
```

PS: Note that as for now there is an inconsistency in the logic of the Weblate's behaviour: you supposed to mark source strings as forbidden, but it's the translations who actually become forbidden. I hope this one will be changed soon because as for now this is quite unmanageable...

Reply:
I think that is the "strict-same" or "read-only". "forbidden" means to avoid some word from the glossary I think? Forgot a "to" in the string above, good catch.

Reply:
yep, as far as I can understand it means that the translation of the terms marked in the glossary as `forbidden` should be avoided. My note was about that even if the flag is applied to the source string, the "forbidden" becomes it's certain translation, which I've tried to highlight in the variant.

Also note that as I can see this feature is unfinished and under active development now, so I'd suggest to wait a bit with improving of the documentation until it settles down a bit as It will be changed every other week...